test_files/class/class.ts

```diff
@@ -117,22 +117,6 @@ let abstractClassVar: AbstractClass;
 abstractClassVar = new ClassImplementsAbstractClass();
 abstractClassVar = new ClassExtendsAbstractClass();
 
-// Reproduce issue #333: type/value namespace collision.
-// Because Zone is both a type and a value, the interface will be dropped
-// when converting to Closure, so the "implements" should be ignored for
-// both the direct use and the use via a typedef.
-interface Zone {
```

Comment:
You're removing coverage here. Can we keep a test that shows what happens when we have a type/value conflict in a regular `.ts` file?